cinema_service/settings.py

```diff
@@ -134,7 +138,7 @@
 STATIC_URL = "static/"
 
 MEDIA_URL = "/media/"
-MEDIA_ROOT = BASE_DIR / "media"
+MEDIA_ROOT = "/files/media"
```

Comment:
The `MEDIA_ROOT` is set to `/files/media`, which is an absolute path. Consider using a path relative to `BASE_DIR` to avoid issues with different environments.